### PR TITLE
Uncalibrated MMR shows as 0 on MMR graph

### PR DESCRIPTION
**Pull request by Argn0**
fixes https://github.com/odota/web/issues/1443

example: https://www.opendota.com/players/129854531/mmr

### CODE REVIEW

File: src/components/Player/Pages/MMR/MMR.jsx [hunk 1]
```diff
@@ -7,6 +7,15 @@ import strings from 'lang';
 import Container from 'components/Container';
 import Info from 'components/Alerts/Info';
 
+const filterNullValues = function (columns) {
```
Comment (howardchung) [inline src/components/Player/Pages/MMR/MMR.jsx]: suggest doing this via .map instead

File: src/components/Player/Pages/MMR/MMR.jsx [hunk 2]
```diff
@@ -17,7 +26,7 @@ const MMRInfo = (
 const MMR = ({ columns, error, loading }) => (
   <div>
     <Container title={strings.heading_mmr} subtitle={MMRInfo} error={error} loading={loading}>
-      <MMRGraph columns={columns} />
+      <MMRGraph columns={filterNullValues(columns)} />
```
Comment (howardchung) [inline src/components/Player/Pages/MMR/MMR.jsx]: actually investigate inside the mmrgraph method, it may make more sense to filter in there.

**Comment by Argn0** [comment]
We could also connect the lines instead of having a gap.

**Comment by howardchung** [comment]
0's not really a valid value, so we shouldn't display it at all (really, we shouldn't even ingest the value in core)

File: src/components/Visualizations/Graph/MMRGraph.jsx [hunk 3]
```diff
@@ -18,8 +18,8 @@ const StyledGraphArea = styled.div`
 user-select: none;
 `;
 
-const filterNullValues = function (columns) {
-  const c = columns;
+const filterZeroValues = function (column) {
```
Comment (howardchung) [inline src/components/Visualizations/Graph/MMRGraph.jsx]: use an arrow function

File: src/components/Visualizations/Graph/MMRGraph.jsx [hunk 4]
```diff
@@ -18,8 +18,8 @@ const StyledGraphArea = styled.div`
...
+  const c = column;
```
Comment (howardchung) [inline src/components/Visualizations/Graph/MMRGraph.jsx]: suggest `return {...column, solo_competitive_rank: c.solo_competitive_rank || null...`

Comment (howardchung) [reply src/components/Visualizations/Graph/MMRGraph.jsx]: don't need the const c = column

**Comment by howardchung** [comment]
lint issue

**Pull request merged**